site/content/docs/5.2/utilities/borders.md

```diff
@@ -49,6 +49,17 @@ Change the border color using utilities built on our theme colors.
 <span class="border border-white"></span>
```

Comment:
The white doesn't have subtle ?

Reply:
Fine with it since this is a color and not a concept. However, missing the black one, see https://github.com/twbs/bootstrap/pull/37905.